README.md

```diff
@@ -84,9 +84,10 @@ $ paru -S dura-git
 
 ### By Source
 
-1. Install Rust (e.g., `brew install rustup`)
-1. Clone this repository
-1. Run `cargo install --path .`
+1. Install Rust (e.g., `brew install rustup && brew install rust`)
+1. Clone this repository (e.g., `git clone git@github.com:tkellogg/dura.git`)
```

Comment:
You want to clone via the https URL otherwise you need to have a ssh key associated with GitHub to clone.

```suggestion
1. Clone this repository (e.g., `git clone https://github.com/tkellogg/dura.git`)
```